impl-endpoints/nss/ech_key_converter.py

```diff
@@ -0,0 +1,73 @@
+import sys
+import struct
+import base64
+
+ECH_VERSION = 0xFE09
+DHKEM_X25519_SHA256 = 0x0020
+pkcs8_start = b"\x30\x67\x02\x01\x00\x30\x14\x06\x07\x2A\x86\x48\xCE\x3D\x02\x01\x06\x09\x2B\x06\x01\x04\x01\xDA\x47\x0F\x01\x04\x4C\x30\x4A\x02\x01\x01\x04\x20"
+pkcs8_oid = b"\xa1\x23\x03\x21\x00"
+
+
+def convert_ech_key(in_file, out_file):
+    with open(in_file, "rb") as f:
+        ech_keys = base64.b64decode(f.read(), None, True)
```

Comment:
Suggest s/ech_keys/ech_keypair/. ech_keys sounds like it's a sequence of ECHKey structures.